src/components/ExternalLink.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {Text} from 'react-native';
+import PropTypes from 'prop-types';
+import openURLInNewTab from '../libs/openURLInNewTab';
+
+const propTypes = {
+    /** External URL we want to link */
+    href: PropTypes.string.isRequired,
+
+    /** Any children to display */
+    children: PropTypes.node,
+};
+
+const defaultProps = {
+    children: null,
+};
+
+const ExternalLink = ({
+    href,
+    children,
+    ...props
+}) => (
+    <Text
+        accessibilityRole="link"
+        onPress={(e) => { openURLInNewTab(href); e.preventDefault(); }}
```

Comment:
This should be split up to only have one statement per line (should probably also prevent default first):

```javascript
onPress={(e) => {
    e.preventDefault();
    openURLInNewTab(href);
})
```